multicluster/apis/multicluster/v1alpha1/labelidentity_types.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2022 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=labelidentities,scope=Cluster
+
+// LabelIdentity is an imported label identity from the ClusterSet.
+// For each unique label identity, a LabelIdentity will be created in the member cluster.
+type LabelIdentity struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec LabelIdentitySpec `json:"spec,omitempty"`
+}
+
+type LabelIdentitySpec struct {
+	// Label is the normalized string of a label identity.
+	// The format of normalized label identity is `namespace:(?P<nslabels>(.)*)&pod:(?P<podlabels>(.)*)`
```

Comment:
It can save some bytes by changing "namespace:" to "ns:". No strong opinion.

Reply:
Done